contracts/tournament/Tournament.cairo

```diff
@@ -221,25 +314,31 @@ end
 @external
 func open_tournament_registration{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
 ) -> (success: felt):
-    Ownable_only_owner()
+    #Ownable_only_owner()
```

Comment:
yes, that's definitely something I did just for testing purpose.
Do you think we should keep it to remember we must add it, or remove it because commented code is prohibited? My heart hesitates

Reply:
I think we should never have commented code :)
You can instead have a `TODO` comment?

Reply:
I agree. No commented line is better. We will enforce that through the CI at some point

Reply:
agreed